src/etc/rc.newipsecdns

```diff
@@ -36,21 +49,20 @@ require_once("auth.inc");
 require_once("ipsec.inc");
 require_once("vpn.inc");
 
-/* make sure to wait until the boot scripts have finished */
-if (file_exists("{$g['varrun_path']}/booting")) {
-	return;
-}
-
-if (ipsec_enabled()) {
-	sleep(15);
-	log_error("IPSEC: One or more IPsec tunnel endpoints has changed its IP. Refreshing.");
-} else {
+if (!ipsec_enabled()) {
+	unlock($ipseclck_pending);
```

Comment:
In the structure it would indeed be somewhat more logical, however calling try_lock is a lot 'cheaper' if it turns out that it can just 'return' than parsing the configuration into memory by the include_once calls below it.

If you want, i can move it around.